WordPress/WordPressTest/WPAnalyticsTrackerAutomatticTracksTests.m

```diff
@@ -0,0 +1,61 @@
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+#import "WPAnalyticsTrackerAutomatticTracks.h"
+#import <Automattic-Tracks-iOS/TracksService.h>
+#import <OCMock/OCMock.h>
+
+@interface WPAnalyticsTrackerAutomatticTracksTests : XCTestCase
+
+@property (nonatomic, strong) WPAnalyticsTrackerAutomatticTracks *subject;
+@property (nonatomic, strong) id serviceMock;
+
+@end
+
+@interface WPAnalyticsTrackerAutomatticTracks (Testing)
+
+@property (nonatomic, strong) TracksService *tracksService;
+@property (nonatomic, strong) TracksContextManager *contextManager;
+
+@end
+
+@implementation WPAnalyticsTrackerAutomatticTracksTests
+
+- (void)setUp {
+    [super setUp];
+
+    self.subject = [[WPAnalyticsTrackerAutomatticTracks alloc] init];
+    self.serviceMock = OCMStrictClassMock([TracksService class]);
+    self.subject.tracksService = self.serviceMock;
+    self.subject.contextManager = nil;
+}
+
+- (void)tearDown {
+    [super tearDown];
+    
+    self.subject = nil;
+    self.serviceMock = nil;
+}
+
+- (void)testVerifyTracksNamesMappings {
+    for (NSUInteger x = 0; x <= WPAnalyticsStatTwoFactorSentSMS; ++x) {
```

Comment:
Yup I like the idea of adding the enum as the last item. I mentioned in the PR but I used the wrong wording.

>  we add an enum in for the last placeholder.

Because this enum comes from the WPAnalytics pod its somewhat out of the scope of the PR for now. I'll log an issue in the right repo :+1:.